hack/test-e2e-provider-local.sh

```diff
@@ -30,7 +30,7 @@ cd $repo_root
 
 version=$(git rev-parse HEAD)
 make docker-images
-docker tag eu.gcr.io/gardener-project/gardener/extensions/networking-calico:latest networking-calico-local:$version
+docker tag europe-docker.pkg.dev/gardener-project/snapshots/extensions/networking-calico:latest networking-calico-local:$version
```

Comment:
Will this work for release build? If we push them to a different path how should this work?